layers/+lang/emacs-lisp/README.org

```diff
@@ -68,8 +68,9 @@ As this state works the same for all files, the documentation is in global
 * Debugging Elisp
 Here is an interactive quick start to debug Emacs Lisp.
 
-First you need to read this file in Emacs in order to have the following code
-block to be interactively opened as Emacs Lisp.
+First you need to read this file in Spacemacs (`SPC h l emacs-lisp RET` and then
```

Comment:
Please don't intermix markdown and org formatting, this is don't use backticks instead use `~` for text formatting. This will be translated correctly for display in github.